roles/jboss_eap/tasks/main.yml

```diff
@@ -18,3 +18,8 @@
         - /var/log/jboss-as
         - /usr/log/jboss-as
       when: '"jboss.eap.common-directories" in facts_to_collect'
+    - name: gather jboss.eap.processes
+      raw: pgrep -f eap
```

Comment:
looks like a new program we need to depend on. How common is this to Linux platforms vs. `ps -f | grep eap`

Reply:
do we know what Wildfly looks like here? So we can differentiate, I know Mark was looking at both product and upstream.

Reply:
I just did some research. Looks like pgrep has been in the procps package since at least 2002 (https://gitlab.com/procps-ng/procps/commits/master/pgrep.c), although I don't know if it's always been packaged in distributions. I'm okay switching to `ps`.

Reply:
I checked the latest Wildfly tar distribution. It has a very similar-looking folder hierarchy, but rooted at a folder named `wildfly-<version>` instead of `jboss-eap-<version>`.

Reply:
I found a file listing for the wildfly RPM. Looks like a similar directory structure to the EAP one, but again, the base directory is wildfly instead of eap7. https://www.rpmfind.net/linux/RPM/fedora/devel/rawhide/x86_64/w/wildfly-10.1.0-11.fc27.noarch.html